run.sh

```diff
@@ -5,8 +5,15 @@ then
     gh api /repos/$GITHUB_REPOSITORY/releases | jq -e -r '.[] | select(.draft == true and .name == "next") | .body' | egrep "$INTERESTING_CATEGORIES"
 fi
 export MAVEN_OPTS=-Djansi.force=true
-mvn -B -V -s $GITHUB_ACTION_PATH/settings.xml -ntp -Dstyle.color=always -Dset.changelist -DaltDeploymentRepository=maven.jenkins-ci.org::default::https://repo.jenkins-ci.org/releases/ -Pquick-build -P\!consume-incrementals clean deploy
-version=$(mvn -B -ntp -Dset.changelist -Dexpression=project.version -q -DforceStdout help:evaluate)
-gh api -F ref=refs/tags/$version -F sha=$GITHUB_SHA /repos/$GITHUB_REPOSITORY/git/refs
+if fgrep -sq changelist.format .mvn/maven.config
+then # JEP-229
+    mvn -B -V -s $GITHUB_ACTION_PATH/settings.xml -ntp -Dstyle.color=always -Dset.changelist -DaltDeploymentRepository=maven.jenkins-ci.org::default::https://repo.jenkins-ci.org/releases/ -Pquick-build -P\!consume-incrementals clean deploy
```

Comment:
https://github.com/jenkins-infra/jenkins-maven-cd-action/pull/14/files?w=1

Reply:
Would it be better to make this a configuration option so that people who missed incrementals don’t accidentally go this route?

Reply:
> people who missed incrementals

Or _did_ incrementalify (e.g. via `archetypes`), but did not set up `changelist.format`. Yeah that is probably wise.